website/docs/pipelines/promoting-applications.mdx

```diff
@@ -4,12 +4,17 @@ hide_title: true
 ---
 
 import TierLabel from "./../_components/TierLabel";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import AlphaWarning from "../_components/_alpha_warning.mdx";
```

Comment:
There's nothing in this document warning about the risks of compromise of the token.

You require "write" to contents access, **and** create pull request access, this means that compromising the token can lead to changing the git repository and creation of legitimate looking changes/pull requests.

Reply:
added, not to the warning but to the pull request promotion strategy section as it seems better suited

https://github.com/weaveworks/weave-gitops/pull/3426/files#diff-b64d18b8c0ecf5a67a9bed187be99a4ed49057ecba3d55bb03de2b9a681165e8R142-R145